src/test/java/me/karakelley/http/server/HttpServerTest.java

```diff
@@ -485,6 +484,31 @@ void test404ForGetRequestAfterResourceDeleted() {
     });
   }
 
+  @Test
+  void testServesHtmlFileForDirectoryWhenPresent() {
```

Comment:
If your computer didn't have copy/paste what do you think the tests in this file would look like? There's so much boilerplate in each one it's hard to tell what's important.

Reply:
👍 Nice job not muddying this PR with those changes.